themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,133 @@
+
+## Using this template
+
+To use this template to deploy an ECS cluster that's running your container service, make sure you've [installed Pulumi]({{< relref "/docs/get-started/install" >}}) and [configured your AWS credentials]({{< relref "/registry/packages/aws/installation-configuration#credentials" >}}), then create a new [project]({{< relref "/docs/intro/concepts/project" >}}) using the template in your language of choice:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```bash
+$ mkdir my-container-service && cd my-container-service
+$ pulumi new container-aws-typescript
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ mkdir my-container-service && cd my-container-service
+$ pulumi new container-aws-python
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ mkdir my-container-service && cd my-container-service
+$ pulumi new container-aws-go
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ mkdir my-container-service && cd my-container-service
+$ pulumi new container-aws-csharp
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```bash
+$ mkdir my-container-service && cd my-container-service
+$ pulumi new container-aws-yaml
+```
+
+{{% /choosable %}}
+
+Follow the prompts to complete the new-project wizard. When it's done, you'll have a complete Pulumi project that's ready to deploy and configured with the most common settings. Feel free to inspect the code in {{< langfile >}} for a closer look.
+
+## Deploying the project
+
+The template requires no additional configuration. Once the new project is created, you can deploy it immediately with [`pulumi up`]({{< relref "/docs/reference/cli/pulumi_up" >}}):
+
+```bash
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) values:
+
+url
+: The HTTP URL for the container's endpoint.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources. The computed `url`, for example, can be used from the command line to open the newly deployed container service in your favorite web browser:
+
+```bash
+$ open $(pulumi stack output url)
+```
+
+## Customizing the project
+
+Projects created with the Container Service template expose the following [configuration]({{< relref "/docs/intro/concepts/config" >}}) settings:
+
+container_port
+: Specifies the port mapping for the container. Defaults to port 80.
+
+cpu
+: Specifies the amount of CPU to use with each task or each container within a task. Defaults to 512.
+
+memory
+: Specifies the amount of memory to use with each task or each container within a task. Defaults to 128.
+
+image
+: The container image that is used to build the application. Defaults to the dockerfile in the `app` folder.
```

Comment:
This technically isn't the container image right? It is the Dockerfile?

Reply:
@cnunciato can you advise on the right wording here?

Reply:
just updated it here, lmk if that works: https://github.com/pulumi/pulumi-hugo/pull/1954/commits/6c5ef01af8514b22633a1287c8517108ae4ea4b5